controllers/homeRoutes.js

```diff
@@ -35,6 +36,28 @@ router.get('/journals', async (req, res) => {
     };
 });
 
+// Specific journal
```

Comment:
Is this route to generate the entries from one specific journal? If so, we already have that

Reply:
No, it's to get the journal itself so the journal title can be populated on the page.

Reply:
But it's also pulling in the entry data, so how is this different from the /journals/:id/entries route? It seems like there's overlap in functionality between these two routes

Reply:
Yeah, the original version only pulled in the journal information, but I realized this was the solution to how to populate the entries. I left the get entries route in the file because I wasn't sure whether there was a use case for that by itself and I didn't want to get rid of it if we still needed it for anything, but my get journal route is how the journal and its entries get populated together.